src/index.tsx

```diff
@@ -33,23 +34,32 @@ function createCanvasInstance<Props extends SketchProps = SketchProps>(
 }
 
 function removeCanvasInstance<Props extends SketchProps = SketchProps>(
-  canvasInstanceRef: MutableRefObject<P5CanvasInstance<Props> | undefined>
+  canvasInstanceRef: React.MutableRefObject<P5CanvasInstance<Props> | undefined>
 ) {
   canvasInstanceRef.current?.remove();
   canvasInstanceRef.current = undefined;
 }
 
-function ReactP5WrapperComponent<Props extends SketchProps = SketchProps>({
+function ReactP5WrapperComponentGuard<Props extends SketchProps = SketchProps>({
   sketch,
-  children,
   ...props
 }: P5WrapperProps<Props>) {
   if (sketch === undefined) {
-    throw new Error("[ReactP5Wrapper] The `sketch` prop is required.");
+    console.error("[ReactP5Wrapper] The `sketch` prop is required.");
```

Comment:
Error would break a users client scripts potentially, error logging and empty render node seems better for DX / UX. WDYT?

Reply:
I think this can be a good idea because it's the fault of the developers, not the users, so even if this error is handled correctly on the client, it doesn't tell the user anything